java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -918,14 +918,10 @@ public void getrange(BaseClient client) {
         assertEquals("", client.getrange(stringKey, -1, -3).get());
 
         // a redis bug, fixed in version 8: https://github.com/redis/redis/issues/13207
-        assertEquals(
-                SERVER_VERSION.isLowerThan("8.0.0") ? "T" : "",
-                client.getrange(stringKey, -200, -100).get());
+        assertEquals("T", client.getrange(stringKey, -200, -100).get());
```

Comment:
Revert this - versions 7 and 8 return different result due to this bug

Reply:
not in valkey